samples/test/utils/clean.js

```diff
@@ -0,0 +1,80 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const {GameServerClustersServiceClient} = require('@google-cloud/game-servers');
+
+/**
+ * Utility function for removing unneeded realms from project.
+ * @param {RealmsServiceClient} realmsClient Realms service client for project
+ * @param {GameServerClustersServiceClient} gameServerClusterClient Game Server service client for project
+ * @param {string} projectId Google Cloud Project ID
+ * @param {string} location project location
+ */
+module.exports = async (
+  realmsClient,
```

Comment:
I kinda suspect you don't need most of these parameters.  
- `realmsClient` - you can instantiate a new one in here
- `gameServerClusterClient` - same - just create a new one
- `projectId` - use the result of `await client.getProjectId()`
- `location` - assuming we hard code this in the tests.  Just use that one!

I suspect that will simplify the invocation of this all over the map by making these assumptions (good ones IMO)

Reply:
Done.